src/frontend/js/components/SearchFilterGroupModal/index.spec.tsx

```diff
@@ -96,6 +96,8 @@ describe('<SearchFilterGroupModal />', () => {
 
       // The modal is rendered
       const openButton = screen.getByText('More options');
+      // Make sure there is a more explicit text for screen readers
+      expect(openButton.querySelector('.offscreen')?.textContent).toEqual('(Universities)');
```

Comment:
Did you try to query this element with `getByRole` ?
```
screen.getByRole('button', { name: 'More options (Universities)' });
```

Reply:
I think I tried but will check again

Reply:
I guess I didn't try correctly because it works :') thanks